testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1203,6 +1203,11 @@ public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
         assertQuery("SELECT row_id FROM " + tableName + " WHERE value <= " + sampleValueLiteral, "VALUES 'sample value'");
         assertQuery("SELECT row_id FROM " + tableName + " WHERE value > " + sampleValueLiteral, "VALUES 'high value'");
         assertQuery("SELECT row_id FROM " + tableName + " WHERE value <= " + highValueLiteral, "VALUES ('sample value'), ('high value')");
+        assertQuery("SELECT row_id FROM " + tableName + " WHERE value < " + highValueLiteral, "VALUES 'sample value'");
+        assertQuery("SELECT row_id FROM " + tableName + " WHERE value >= " + highValueLiteral, "VALUES 'high value'");
+        assertQuery("SELECT row_id FROM " + tableName + " WHERE value > " + highValueLiteral, "SELECT '' WHERE false");
+        assertQuery("SELECT row_id FROM " + tableName + " WHERE value <= " + sampleValueLiteral, "VALUES 'sample value'");
```

Comment:
this one is already above

Reply:
This adds a lot to an already long test. Maybe we shouyld rather have a dedicated test method testing for case sensitiveity? I think this issue is well isolated to char and varchar types.